### PR TITLE
fix regression - highlight sidebar

### PR DESCRIPTION
**Pull request by sravfeyn**
fixes regression (if you go to any report/export page, it's not highlighted in the sidebar) introduced by [this](https://github.com/dimagi/commcare-hq/commit/ad70b65e2d19ca7951fed0c07f75065633dc9505#diff-e16d894b866f291a81f5ce256e1bd162L648)

### CODE REVIEW

**Comment by dimagimon** [comment]
:+1: No lint errors found.

File: corehq/apps/hqwebapp/templatetags/menu_tags.py [hunk 1]
```diff
@@ -120,7 +120,7 @@ def format_sidebar(context):
         # and see if the nav needs a subnav for the current contextual item
         for section_title, navs in sections:
             for nav in navs:
-                if request.get_full_path().startswith(nav['url']):
+                if request.get_full_path().startswith(nav['url']) or request.build_absolute_uri() == nav['url']:
```
Comment (czue) [inline corehq/apps/hqwebapp/templatetags/menu_tags.py]: should the second check also be `startsWith`?

Comment (sravfeyn) [inline corehq/apps/hqwebapp/templatetags/menu_tags.py]: oh yes to discard params.

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Pull request merged**